### PR TITLE
Implement adding image feature

### LINKED ISSUE #46: Implement adding image feature in blog
- User should be able to insert the picture to Blogs

### PR DESCRIPTION
**Pull request by LevNut**
- image can be uploaded during add blog
- image can be edited
- adjust setting
- requirement: pip install Pillow

### CODE REVIEW

**Comment by Bheem6005** [comment]
+1 image can auto resize, add image button in both create and edit blog page

File: kuhub/templates/kuhub/create_blog.html [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 {% block content %}
+
 <div class="col-md-8">
     <div class="card mb-4 mt-4">
         <div class="card-header">
             Create Blog Post
         </div>
         <div class="card-body">
-            <form method="POST">
+            <form method="POST" enctype="multipart/form-data">
```
Comment (NarawishS) [inline kuhub/templates/kuhub/create_blog.html:11]: what is entype?

Comment (LevNut) [reply kuhub/templates/kuhub/create_blog.html:11]: Enctype is a thing that used to specifies the way that forms-data should be encoded before sending it to server. To add on, multipart/forms-data is one of a method that is used to include an input file type.

**Comment by NarawishS** [comment]
template look bad, what the purpose of this implement?
is it to add image to bottom of the blog? or is it to show in index page?

**Comment by LevNut** [comment]
@NarawishS Oh sorry about that I forgot to show the image at index page, I will fix this issues thanks for the comment

**Pull request merged**